bundle/edu.gemini.pot/src/test/scala/edu/gemini/spModel/target/env/Arbitraries.scala

```diff
@@ -159,7 +160,153 @@ trait Arbitraries extends edu.gemini.spModel.core.Arbitraries {
   implicit val arbSingleAsterism: Arbitrary[Asterism.Single] =
     Arbitrary(arbitrary[SPTarget].map(Asterism.Single(_)))
 
-  implicit val arbAsterism: Arbitrary[Asterism] =
-    Arbitrary(arbitrary[Asterism.Single]) // TODO:ASTERISM: add GHOST
 
+  // Seal off all the generators for the pieces of GhostAsterisms.
+  private object GhostGens {
+    import GhostAsterism._
+    import GhostAsterism.GhostStandardResTargets._
+
+    // This generates a list of n coordinates, wherein the last n-1 are within distance [lower, upper) from the first.
+    def genNCoordsWithinDistance(n: Int, lower: Double, upper: Double): Gen[List[Coordinates]] =
+      for {
+        c  <- arbitrary[Coordinates]
+        cs <- listOfN(n-1, genCoordsWithinDistance(c, lower, upper))
+      } yield c :: cs
+
+    // TODO:GHOST Change these to GHOST constants as we learn more, and move to Ghost, probably.
+    val inRangeDistance: Double =   5.0 // in arcsecs
+    val border:          Double = 100.0 // in arcsecs
+    val borderDelta:     Double =   5.0 // in arcsecs
```

Comment:
Yes, that's what I think border should be. I definitely think it should be moved to the `Ghost` component, but I shoved it in here since we don't know the values or how we are going to represent them yet. I have no objections moving them to `Ghost` now, but based on your other comment, it seems we might need some rethinking about this.